### PR TITLE
Redact README

### PR DESCRIPTION
**Pull request by wojtodzio**
A few language/stylistics fixes 📝
Hope you like it!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,14 +140,14 @@ emptyPost.favorites(payload).then((favoritesPosts) => {
...
 
-* `git clone <repository-url>` this repository
+* `git clone https://github.com/Exelord/ember-custom-actions.git`
 * `cd ember-custom-actions`
 * `npm install`
 * `bower install`
 
 ## Running
 
-* `ember serve`
```
Comment (Exelord) [inline README.md]: That was correct :)

File: README.md [hunk 2]
```diff
@@ -17,7 +18,7 @@ Ember Custom Actions is compatible with Ember 2.8 (and higher) applications..
 
 ### Configuration
 
-You can define your custom default options in your `config/environment.js` file.
+You can define your custom options in your `config/environment.js` file
```
Comment (Exelord) [inline README.md]: What about:
```
You can define your default options in your `config/environment.js` file.
```

File: README.md [hunk 3]
```diff
@@ -4,8 +4,9 @@
   <a href='https://travis-ci.org/Exelord/ember-custom-actions'><img src="https://travis-ci.org/Exelord/ember-custom-actions.svg?branch=master" alt="Dependency Status" /></a> <a href='https://gemnasium.com/github.com/Exelord/ember-custom-actions'><img src="https://gemnasium.com/badges/github.com/Exelord/ember-custom-actions.svg" alt="Dependency Status" /></a>
 </p>
 
-The Ember add-on lets you call custom actions to your JSON API server.
-Ember Custom Actions is compatible with Ember 2.8 (and higher) applications..
+Ember add-on, which lets you call custom actions to your JSON API server.
```
Comment (Exelord) [inline README.md]: I would change this paragraph to:

```md
Ember Custom Actions is a package for defining custom API actions, dedicated for Ember 2.8 (and higher) applications.
```

Comment (wojtodzio) [inline README.md]: Great 👍

File: README.md (hunk 2 above)
Comment (wojtodzio) [inline README.md]: I suppose this is more a philosophical question, but Isn't it that this add-on defines **default** options (inside `addon/config.js`), and by doing this `module.exports` you're in fact overriding them with your **custom** ones?

File: README.md (hunk 1 above)
Comment (wojtodzio) [inline README.md]: 🤣 Fixing it right away

**Comment by wojtodzio** [comment]
@Exelord Done to all but one, which I'm not sure about.
BTW, would you like to change something more by yourself in this PR, feel free to append your commits - as I'm allowing for edits from maintainers to my PR's, you have a write access to the master branch of my fork :) Enjoy!

**Pull request closed without merging**